crates/matrix-sdk/src/oidc/impl/mod.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 The Matrix.org Foundation C.I.C.
```

Comment:
I find the `impl` module that needs `r#` syntax to be referred to pretty ugly. Do we need a sub-module for these 75 LOC at all?

Reply:
Renamed backend/mod.rs. Navigating is pretty fast with LSP, so I'm not too worried about proliferation of files (also the `mod.rs` of OIDC is 1500 lines long, so any attempt to reduce its size is good, IMO)

Reply:
Yeah, I wasn't really concerned about nesting, more about `r#impl`.